src/pages/Onboard/NameTeam.vue

```diff
@@ -165,6 +245,15 @@ export default {
           <v-col v-if="revealNote" key="name" cols="12" class="pb-0">
             <div class="display-1 text-center">
               Let's start by creating your team
+              <v-tooltip bottom>
```

Comment:
I feel like it might be useful to have more information in here.  Maybe something like: 

All Prefect users are required to have a personal team for sandbox use. You can always switch between teams using the side menu.  For more information about teams in Prefect, check out our docs. (With link to docs)

If we include the link, a v-menu (such as we use for concurrency slots on the dashboard) might work better than v-tooltip.

Reply:
^ This one's a suggestion

Reply:
I like it!